plugins/offboard/offboard.h

```diff
@@ -81,6 +92,42 @@ class Offboard : public PluginBase {
                           above). */
     };
 
+    /**
+     * @brief Type for Velocity commands in NE (North East) and Altitude in N (Down) coordinate and
+     * yaw.
+     */
+    struct VelocityAltitudeNEDYaw {
+        float north_m_s; /**< @brief Velocity North in meters/second. */
+        float east_m_s; /**< @brief Velocity East in meters/second. */
+        float down_m; /**< @brief Altitude Down in meters. */
```

Comment:
I find `Altitude Down` confusing but I don't have a better idea.

Reply:
since i called the other two members `velocity`, I can call the third one `position down`